pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -32,26 +32,20 @@ makeWrapper() {
     for ((n = 2; n < ${#params[*]}; n += 1)); do
         p="${params[$n]}"
 
-        if test "$p" = "--set"; then
+        if [[ "$p" == "--set" ]]; then
```

Comment:
Ok Bash 3.2 is not *that* bad. The issue was not case but fallthrough, and or-patterns can be used instead (and in fact are a better solution).
```bash
# Works in bash 3.2
case n in
    p|n) echo yup;;
esac
```